.circleci/config.yml

```diff
@@ -138,25 +138,55 @@ commands:
       - run:
           command: git submodule update -f --init --recursive
 
-  deploy-to-staging-command:
-    description: "Deploy API and UI to staging App Engine"
+  deploy-command:
+    description: "Deploy API and UI to App Engine"
+    parameters:
+      project_name:
+        type: string
+      service_account:
+        type: string
+      credentials_env_var:
+        type: string
+      key_env_var:
+        type: string
     steps:
       - checkout-code
       - attach_workspace:
           at: .
-      - gcloud-auth-login
-      - run:
+      - gcloud-auth-login:
+          credentials_env_var: << parameters.credentials_env_var >>
+          key_env_var: << parameters.key_env_var >>
+      - deploy:
           working_directory: ~/workbench/deploy
           command: |
             ./project.rb deploy \
-              --project all-of-us-rw-staging \
-              --account << pipeline.parameters.circle-service-account >> \
+              --project << parameters.project_name >> \
+              --account << parameters.service_account >> \
               --git-version "${CIRCLE_TAG}" \
               --app-version "${CIRCLE_TAG}" \
               --circle-url "https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}" \
               --key-file ~/workbench/api/circle-sa-key.json \
               --promote
-          name: Deploy API and UI to all-of-us-rw-staging App Engine
+          name: Deploy API and UI to << parameters.project_name >> App Engine
+
+  deploy-to-staging-command:
+    description: "Deploy to staging environment"
+    steps:
+      - deploy-command:
+          project_name: "all-of-us-rw-staging"
+          service_account: "<< pipeline.parameters.circle-service-account >>"
+          credentials_env_var: "GCLOUD_CREDENTIALS"
+          key_env_var: "GCLOUD_CREDENTIALS_KEY"
```

Comment:
(copied from https://github.com/all-of-us/workbench/pull/8224)

I created two new CircleCI secrets containing the values for the stable deploy service account, using the same method we use for GCLOUD_CREDENTIALS and GCLOUD_CREDENTIALS_KEY as described in the [oncall playbook](https://docs.google.com/document/d/1hbYNWzdxuRybTfW3M70hWZfr_TXRVd5rOMR9kZtwAFY/edit?tab=t.0#heading=h.676040liklzd). Once this is merged, I'll add a new section to that playbook.